### ISSUE TITLE
Accessibility 2.4.4 - Link Purpose

### ISSUE THREAD

**Issue by phucduong86** [CodingTrain/website#1229]
No discernible text available for Home page link (logo background). Recommend adding visually hidden text or adding "Home" link in the navigation section.

Results for: (https://thecodingtrain.com/Tutorials/1-p5js-basics/1.3-basics-of-drawing.html)
Outcome: Failed
Findings: link in the youtube video frame's text is not descriptive - suggest changing to "The Coding Train You tube Channel" instead of "Photo image of "
```
<a class="ytp-title-channel-logo" target="_blank" href="https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw" aria-label="Photo image of " style="background-image: url(&quot;https://yt3.ggpht.com/-1rYs64e24jQ/AAAAAAAAAAI/AAAAAAAAAAA/B2LPB2yW030/s68-c-k-no-mo-rj-c0xffffff/photo.jpg&quot;);"></a>
```

The previous/next button must have discernable text alternative (NVDA screen reader only reads "Link" at the moment). Recommend adding aria-label
```
<a href="/Tutorials/1-p5js-basics/1.2-p5js-web-editor.html" class="jump-control ">❰</a>
<a href="/Tutorials/1-p5js-basics/1.4-color.html" class="jump-control ">❱</a>
```
